electron/pandoc/export.ts

```diff
@@ -226,7 +226,7 @@ const mergeAndValidate = (docMeta: Meta, extMeta: Meta, outputPath?: string, toC
     out.metadata = {};
   }
   if (docMeta.mainfont === undefined) {
-    out.metadata.mainfont = '-apple-system, BlinkMacSystemFont, Segoe UI, sans-serif';
+    out.metadata.mainfont = extMeta.mainfont || '-apple-system, BlinkMacSystemFont, Segoe UI, sans-serif';
   }
```

Comment:
Thanks! that indeed seems to be bug... If I remember correctly, I added this line so that the HTML export via pandoc uses the same font as the preview. These fonts won't ever work with `pdflatex`... so probably the better fix would be the following?

```js
  if (docMeta.mainfont === undefined && toFormat === 'html') {
    out.metadata.mainfont = '-apple-system, BlinkMacSystemFont, Segoe UI, sans-serif';
  }
``

Reply:
I'll try.

Reply:
And maybe the background color in the following lines can also be skipped for non-HTML?

Reply:
Done. It works.

Reply:
great, thanks!